### DISCUSSION TITLE
The proxy is enabled, but the server is not connected.

### DISCUSSION THREAD

**Discussion by shuairongzeng** [Ylianst/MeshCentral#6204] [Issues]
Hi everyone, can you help me? I have a problem, the agent is installed and running normally, but the server does not have this device, what should I do?As shown in the picture：
![image](https://github.com/Ylianst/MeshCentral/assets/59594781/e0f7f5c5-9d11-4dcf-b097-dfed95f8767c)
I am using Alibaba Cloud Server, and the port is opened as follows 
![image](https://github.com/Ylianst/MeshCentral/assets/59594781/91c3148a-6db7-44ee-80bf-bb9578e91da6)
![image](https://github.com/Ylianst/MeshCentral/assets/59594781/d4644be2-49d0-49af-ad8b-5823528b514d)
config.json
![image](https://github.com/Ylianst/MeshCentral/assets/59594781/ff3ef7a1-9aaf-4702-b786-844c6a73468a)

**Comment by si458** [comment]
It shows local in the meshagent, which means lan mode, so it will only work when your computers are on the same network as your meshcentral server 

You need to uncomment the cert value in config.json and enter in a DNS NAME that is setup with ur vps ip address, then restart meshcentral

DO NOT USE AN IP ADDRESS AS IP ADDRESSES CAN CHANGE, ALWAYS USE A DNS NAME!

P.s. After the change, u need to uninstall the meshagent and redownload the meshagain again for it to detect ur dns name
